index.html

```diff
@@ -1,54 +1,94 @@
 <!DOCTYPE html>
-<!-- talked about this is class - specifies english language -->
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <title>Portfolio</title>
-    <!-- meta tag for mobile first design -->
     <meta name="viewport" content="width=device-width, initial-scale= 1">
-    <!-- link to font awesome -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="./styles/base.css">
-    <link rel="stylesheet" href="./styles/layout.css">
-    <link rel="stylesheet" href="./styles/modules.css">
+    <link rel="stylesheet" href="vendors/styles/normalize.css">
+    <link rel="stylesheet" href="vendors/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+
+    <script id="project-template" type="text/x-handlebars-template">
+      <project data-language="{{language}}" data-title="{{title}}">
+        <header>
+          <h1>{{title}}</h1>
+            <div class="tagline">
+              Title <address><a href="{{projectUrl}}">{{title}}</a></address>
+              Completed <time pubdate datetime="{{completedOn}}">{{daysAgo}} days ago</time>
+            </div>
+        </header>
+          <section class="project-description">{{description}}</section>
+          <a href="#" class="read-on">Read on &rarr;</a>
+          <hr>
+        </article>
+    </script>
   </head>
+
   <body>
     <header class ="portfolio-header clearfix">
-      <section class="main-header-group">
-        <h1>Anna Shelby</h1>
+      <section class="heading-group-h1">
+        <h1>
+          Anna Shelby
+        </h1>
         <h2>Online Portfolio</h2>
       </section>
       <nav class="main-nav">
-        <!-- hamburger menu font icon added here from font awesome -->
-        <i class="fa fa-bars" aria-hidden="true"></i>
+        <div class="icon-menu"></div>
         <ul>
-          <!-- find icons to use for nav-->
-          <li class="tab" data-content="home">Home</li>
-          <!-- add link to github repo here -->
-          <li class="tab" data-content="projects">Projects</a></li>
-          <li class="tab" data-content="skills">Skills</li>
-          <li class="tab" data-content="about">About</li>
+          <li class="tab" data-content="home"><a href="#" class="icon-home">Home</a></li>
```

Comment:
Some CSS is going on here that these icons are hidden until I hover my mouse over them. Please try to fix that so they're visible.